test/role.manager.spec.js

```diff
@@ -0,0 +1,7 @@
+import chaiHttp from 'chai-http';
+import chai, { expect } from 'chai';
+import { config } from 'dotenv';
+
+config();
+
+chai.use(chaiHttp);
```

Comment:
Add a newline here @Johnpeace.

Use this setting in your VSCode `settings.json` file:
` "files.insertFinalNewline": true,`

Reply:
done